integration-tests/tests/src/utils/import-app.ts

```diff
@@ -33,52 +34,142 @@ function getUrls() {
   };
 }
 
+function getIsAppImporterRemote() {
+  const { isAppImporterRemote } = environment;
```

Comment:
Why wrap this in a function? I would simply put this line to the top of the file.